### PR TITLE
Cu 82pde0 css integration boxed link

### PR DESCRIPTION
**Pull request by Hamza-A**
This branch aims to display the identities on the logins page in a new box stylized as on the wireframe.

## Description
To do that, I had to create 3 new components and their stories:
- `<IdentityContainer />` which will handle all the identities of the same type and replaces the old `<IdentitiesBox />`
- `<BoxedLink />`which contains a single identity and will redirect to its overview on click, it replaces the old `<IdentityBox />`
- `<Separator />` which will mark the end of each `<BoxedLink />` only if the identities' list of the same type > 1

## How Has This Been Tested?
I only had to change an import on a test file

## Screenshots
image

## Types of changes

- Chore (non-breaking change which refactors / improves the existing code base)

## Checklist:

- :green_circle: My code follows the code style of this project.
- :red_circle: My change requires a change to the documentation.
- :red_circle: I have updated the documentation accordingly.
- :green_circle: I have read the [**CONTRIBUTING**][contributing_file] document.
- :red_circle: I have added tests to cover my changes.
- :green_circle: All new and existing tests passed.

[contributing_file]: https://github.com/fewlinesco/connect/blob/master/CONTRIBUTING.md

### CODE REVIEW

**Review by Fuzznimp — APPROVED** [review]

File: src/components/display/fewlines/BoxedLink.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  );
+};
+
+const Box = style.div`
+    height: 7.2rem;
+    margin: 0 ${({ theme }) => theme.spaces.component.xs};
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+`;
```
Comment (Fuzznimp) [inline src/components/display/fewlines/BoxedLink.tsx:21]: You need to remove text decoration for the anchor, so basically:

```suggestion
const Box = style.div`
    height: 7.2rem;
    margin: 0 ${({ theme }) => theme.spaces.component.xs};
    display: flex;
    align-items: center;
    justify-content: space-between;
    text-decoration: none;
`;
```

I don't remember if this includes the "already clicked" link style (font color purple), you'll have to check. Let me know if this part is not clear.

Comment (Fuzznimp) [reply src/components/display/fewlines/BoxedLink.tsx:21]: You also need to handle having the primary identity font in bold, so basically pass it as an argument to `<Box/>`, and if `true`, increase the font weight.

Comment (Hamza-A) [reply src/components/display/fewlines/BoxedLink.tsx]: OK fixed e5e8379 8656a9d
image

File: src/components/display/fewlines/BoxedLink.stories.tsx [hunk 2]
```diff
@@ -5,5 +5,11 @@ import { BoxedLink } from "./BoxedLink";
 export default { title: "BoxedLink", component: BoxedLink };
 
 export const StandardBoxedLink = (): JSX.Element => {
-  return <BoxedLink value={"emailaddress@mail.test"} />;
+  return (
+    <BoxedLink
+      value={"emailaddress@mail.test"}
+      primary={true}
+      status={"validated"}
```
Comment (Fuzznimp) [inline src/components/display/fewlines/BoxedLink.stories.tsx:12]:
```suggestion
      status="validated"
```

File: src/components/display/fewlines/BoxedLink.tsx [hunk 3]
```diff
@@ -3,10 +3,24 @@ import style from "styled-components";
...
+  let className;
+  if (primary) {
+    className = "bold";
+  } else {
+    if (status === "validated") {
+      className = "normal";
+    } else {
+      className = "lighter";
+    }
+  }
```
Comment (Fuzznimp) [inline src/components/display/fewlines/BoxedLink.tsx:20]:
```suggestion
  let className = "lighter";

  if (primary) {
  	className = "bold";
  } else if (status === "validated") {
    className = "normal";
  }
```

File: src/pages/account/logins/index.tsx [hunk 4]
```diff
@@ -62,6 +62,7 @@ const Logins: React.FC<LoginsProps> = ({ sortedIdentities }) => {
       <br />
       <IdentitySection>
         <h3>Email addresses</h3>
+        <br />
```
Comment (Fuzznimp) [inline src/pages/account/logins/index.tsx:65]: You should remove all the `<br />` and instead do:

```css
...
h3 {
  margin-bottom: ...
}
```

**Review by Fuzznimp — CHANGES_REQUESTED** [review]

File: src/components/display/fewlines/BoxedLink.tsx (hunk 3 above)
Comment (Hamza-A) [reply src/components/display/fewlines/BoxedLink.tsx]: OK fixed 4823f7e

File: src/components/display/fewlines/BoxedLink.stories.tsx (hunk 2 above)
Comment (Hamza-A) [reply src/components/display/fewlines/BoxedLink.stories.tsx]: OK fixed 6cfaf5e

File: src/pages/account/logins/index.tsx (hunk 4 above)
Comment (Hamza-A) [reply src/pages/account/logins/index.tsx]: OK fixed 16506b1

**Review by Fuzznimp — APPROVED** [review]

**Pull request merged**